etl/resources/output_config.py

```diff
@@ -17,9 +17,12 @@ class Parsed:
         A dataclass that contains the output directory Path of the ETL.
 
         Properties of Parsed include:
+        - openai_embeddings_directory_path: the output directory Path for data on OpenAI embeddings.
```

Comment:
Should be able to add these as docstrings above the fields themselves. Then the IDE should pick up that the docstring is documenting the field.